<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="4" errors="2" failures="1">
	<testsuite name="package/name1" tests="1" failures="0" errors="1" id="0" hostname="hostname" time="0.141" timestamp="2022-01-01T00:00:00Z">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
		<testcase name="Failure" classname="package/name1" time="0.000">
			<error message="Runtime error"><![CDATA[running tests for  package/name1]]></error>
		</testcase>
	</testsuite>
	<testsuite name="package/name1" tests="2" failures="1" errors="0" id="1" hostname="hostname" time="0.151" timestamp="2022-01-01T00:00:00Z">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
		<testcase name="TestOne" classname="package/name1" time="0.020">
			<failure message="Failed"></failure>
		</testcase>
		<testcase name="TestTwo" classname="package/name1" time="0.130"></testcase>
		<system-out><![CDATA[running tests for  package/name1
	file_test.go:11: Error message
	file_test.go:11: Longer
		error
		message.]]></system-out>
	</testsuite>
	<testsuite name="package/name1" tests="1" failures="0" errors="1" id="2" hostname="hostname" time="0.183" timestamp="2022-01-01T00:00:00Z">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
		<testcase name="Failure" classname="package/name1" time="0.000">
			<error message="Runtime error"><![CDATA[running tests for  package/name1]]></error>
		</testcase>
	</testsuite>
</testsuites>
